<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>755</width>
    <height>683</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton_instructions">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>221</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tekton Pro</family>
      <pointsize>22</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>instructions</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_readme">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>221</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tekton Pro</family>
      <pointsize>22</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>readme</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_begin">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>180</y>
      <width>221</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tekton Pro</family>
      <pointsize>22</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>begin</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_easy">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>280</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tekton Pro</family>
      <pointsize>30</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>easy</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_medium">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>380</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tekton Pro</family>
      <pointsize>30</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>medium</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_hard">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>470</y>
      <width>85</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tekton Pro</family>
      <pointsize>30</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>hard</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_custom">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>270</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tekton Pro</family>
      <pointsize>30</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>custom</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_height">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>320</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tekton Pro</family>
      <pointsize>22</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_width">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>370</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tekton Pro</family>
      <pointsize>22</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>330</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tekton Pro</family>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="text">
     <string>height</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>380</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tekton Pro</family>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="text">
     <string>width</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>430</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tekton Pro</family>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="text">
     <string>mines</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_mines">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>420</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tekton Pro</family>
      <pointsize>22</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>320</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tekton Pro</family>
     </font>
    </property>
    <property name="text">
     <string>height [9] width [9] mines [10]</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>420</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tekton Pro</family>
     </font>
    </property>
    <property name="text">
     <string>height [16] width [16] mines [40]</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>510</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tekton Pro</family>
     </font>
    </property>
    <property name="text">
     <string>height [16] width [30] mines [99]</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>755</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menumenu">
    <property name="title">
     <string>menu</string>
    </property>
    <addaction name="actionmain_menu"/>
    <addaction name="separator"/>
    <addaction name="actioninstructions"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menusettings">
    <property name="title">
     <string>settings</string>
    </property>
    <widget class="QMenu" name="menucustom">
     <property name="title">
      <string>custom</string>
     </property>
     <addaction name="actionwidth"/>
     <addaction name="actionwidth_2"/>
     <addaction name="actionmines"/>
    </widget>
    <addaction name="actioneasy"/>
    <addaction name="separator"/>
    <addaction name="actionmedium"/>
    <addaction name="separator"/>
    <addaction name="actionhard"/>
    <addaction name="separator"/>
    <addaction name="menucustom"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menumenu"/>
   <addaction name="menusettings"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actioneasy">
   <property name="text">
    <string>easy</string>
   </property>
  </action>
  <action name="actionmedium">
   <property name="text">
    <string>medium</string>
   </property>
  </action>
  <action name="actionhard">
   <property name="text">
    <string>hard</string>
   </property>
  </action>
  <action name="actionwidth">
   <property name="text">
    <string>height</string>
   </property>
  </action>
  <action name="actionwidth_2">
   <property name="text">
    <string>width</string>
   </property>
  </action>
  <action name="actionmines">
   <property name="text">
    <string>mines</string>
   </property>
  </action>
  <action name="actionmain_menu">
   <property name="text">
    <string>main menu</string>
   </property>
  </action>
  <action name="actioninstructions">
   <property name="text">
    <string>instructions</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
